pymepixviewer/main.py

```diff
@@ -135,7 +136,15 @@ def __init__(self, timepix_ip, parent=None):
 
     def switchToMode(self):
         self._timepix.stop()
-        if self._current_mode is ViewerMode.TOA:
+        if True: #self._current_mode is ViewerMode.ViewerMode.Trig:
```

Comment:
Maybe this `if` should be removed as it is always `true` or actually make this the `else`.

Reply:
It is outdated.